Automattic/jetpack
# AI Extension: show loading message while the first form field is not present

Fixes #32299.

## Proposed changes:

* When the completion starts, if there is no valid inner blocks to add yet, use a generic, loading message paragraph

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor, add a form block and ask for some form
* After the form is ready, ask for some change on the form
* Before this change, you would see the JP Form placeholder until the first field is ready (see issue for details)
* After this change, you will see a "Loading, please wait..." message until the first field is ready:

<img width="600" alt="Screenshot 2023-08-11 at 18 44 57" src="https://github.com/Automattic/jetpack/assets/6760046/b36b1c7c-ef09-47cc-9ec4-c32e918f324a">
